src/orquestra/sdk/_base/_traversal.py

```diff
@@ -393,9 +393,10 @@ def _make_import_model(imp: _dsl.Import):
             id=id_,
         )
     elif isinstance(imp, _dsl.GitImport):
+        repo_url = _git_url_utils.parse_git_url(imp.repo_url)
         return ir.GitImport(
             id=id_,
-            repo_url=imp.repo_url,
+            repo_url=repo_url,
```

Comment:
It worked because of the magic backwards compat:

https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/09f4e065602afa204d16e4b3c8a043b17afc15eb/src/orquestra/sdk/schema/ir.py#L59-L68

Reply:
I did not test it live, but as @jamesclark-Zapata it actually doesnt change much, its already used like that but with some pydantic-magic that pyright has no chance to catch